src/Fixer/PhpUnit/PhpUnitNoExpectationAnnotationFixer.php

```diff
@@ -275,12 +276,12 @@ private function extractContentFromAnnotation(Annotation $annotation)
     private function annotationsToParamList(array $annotations)
     {
         $params = [];
-        $exceptionClass = '\\'.ltrim($annotations['expectedException'], '\\');
+        $exceptionClass = ltrim($annotations['expectedException'], '\\');
 
         if ($this->configuration['use_class_const']) {
-            $params[] = $exceptionClass.'::class';
+            $params[] = '\\'.$exceptionClass.'::class';
         } else {
-            $params[] = "'{$exceptionClass}'";
+            $params[] = "'${exceptionClass}'";
```

Comment:
This change should be reverted as well.

Reply:
@julienfalque This is actually what PHP-CS-Fixer applied after I have changed that line. I also was puzzled by this, could it be that the fixer suggests a different bracket format if there is no bracket format in use?

I removed it from the change-set in any case, so please consider your comment addressed.